tests/components/MapboxMap.stories.tsx

```diff
@@ -20,7 +21,13 @@ const meta: ComponentMeta<typeof MapboxMap> = {
       control: false,
```

Comment:
nit: can we also remove the controls for `getCoordinate` and `onDrag`? we don't want people to be able to pass in functions or components through the Storybook site